execution/scan/literal_selector.go

```diff
@@ -27,10 +27,11 @@ type numberLiteralSelector struct {
 	once        sync.Once
 
 	val float64
+	t   model.OperatorTelemetry
```

Comment:
Don't forget to embed this directly into the operator - this would remove some code duplication:

```suggestion
	model.OperatorTelemetry
```